db/warehouse/migrate/20220208180300_indexes_to2022_tables.rb

```diff
@@ -0,0 +1,12 @@
+class IndexesTo2022Tables < ActiveRecord::Migration[6.1]
+  def change
+    spec_version = '2022'
```

Comment:
Dead code?

Reply:
yep, I had intentions of using it, but didn't need to.